app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  validates_presence_of :email, :password, :full_name
+  validates_presence_of :email 
+
+  has_secure_password
```

Comment:
this should be `validations: false` otherwise i think it requires `password_confirmation`